### DISCUSSION TITLE
Add Youtube playlist

### DISCUSSION THREAD

**Discussion by pmmontanari** [rekkyrosso/ampcast#13] [Ideas]
Title says it all: I don't know how difficult it is, or even if it is viable (or too specific/niche to be worth implementing), but having the ability to pass a Youtube playlist URL to fetch every song of the playlist would be nice

I am trying to move my playlists from Apple Music to Youtube (not Youtube Music), and I was checking if I would be able to import them into ampcast (by passing their url) to continue with my normal use case.

Thanks!

**Comment by rekkyrosso** [comment]
If you register a YouTube client using the desktop app or via self-hosting then you can interact with the YouTube API. That gives you the same playlist access as the other streaming services that ampcast supports.

Ampcast used to support playlist loading if you weren't using the API. However, it involved spamming Google with lots of requests that might not ultimately result in play requests. So I removed that functionality but I still have the code.
